db/write_batch.cc

```diff
@@ -167,9 +172,8 @@ class TimestampAssigner : public WriteBatch::Handler {
   }
 
   Status DeleteRangeCF(uint32_t, const Slice& begin_key,
-                       const Slice& end_key) override {
+                       const Slice& /* end_key */) override {
     AssignTimestamp(begin_key);
-    AssignTimestamp(end_key);
```

Comment:
This is sort of out of scope, but it seems like a mistake to assign two timestamps for start and end key. The end key is actually stored in the value when KVs are encoded. It became relevant because we can't support such an update to a part of the value in our `ProtectionInfo.*` classes.